uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-widget/leave-widget-next-leave.component.js

```diff
@@ -72,8 +74,8 @@ define([
      * @return {Boolean}
      */
     function bindingsAreReady () {
-      return vm.contactId && vm.leaveRequestStatuses &&
-        vm.leaveRequestStatuses.length;
+      return vm.absenceTypes && vm.absenceTypes.length && vm.contactId &&
+        vm.leaveRequestStatuses && vm.leaveRequestStatuses.length;
```

Comment:
I can see the previous versions had `vm.leaveRequestStatuses.length` and now it is simply `vm.leaveRequestStatuses`. Are you sure this is not a mistake?

Reply:
Apologies.